data_structures/arrays/sprialmatrix.py

```diff
@@ -0,0 +1,54 @@
+def spiralorder(matrix: list[list[int]]) -> list[int]:
+    """
+    Get the elements of a 2D matrix in spiral order.
+
+    Args:
+    matrix (list[list[int]]): The input 2D matrix.
+
+    Returns:
+    List[int]: A list containing the elements of the matrix in spiral order.
+
+    Example:
+    >>> a = [[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12], [13, 14, 15, 16]]
+    >>> spiralOrder(a)
+    [1, 2, 3, 4, 8, 12, 16, 15, 14, 13, 9, 5, 6, 7, 11, 10]
```

Comment:
Have you tried with python -m doctest -v data_structures/arrays/sprialmatrix.py

Reply:
yes if build is successful then remaining two are not successful